src/AdminConsentRequiredController.js

```diff
@@ -59,6 +59,7 @@ const ConsentHeader = View.extend({
 
 const ConsentForm = {
   noCancelButton: false,
+  showCancelBeforeSave: true,
```

Comment:
added a comment on your courage jira..  I feel this is too specific for this use case and may not be a good candidate for courage change..